src/main/java/uk/gov/beis/els/categories/televisions/controller/TelevisionController.java

```diff
@@ -4,10 +4,8 @@
 
```

Comment:
PR already done for this file in [ELG-6](https://github.com/UKGovernmentBEIS/energy-label-service/pull/87)